frontend/cypress/integration/featureFiles/graph_toolbar.feature

```diff
@@ -0,0 +1,107 @@
+Feature: Kiali Graph page - Toolbar (various)
+
+  User opens the Graph page and manipulates the "error-rates" demo via Toolbar.
+
+  Background:
+    Given user is at administrator perspective
+
+# NOTE: Graph Display menu has its own test script
+#       - tests empty graph as well
+# NOTE: Graph Find/Hide has its own test script
+# NOTE: Graph Replay has its own test script
+
+@graph-page-toolbar
+Scenario: Graph alpha namespace with query params
+  When user graphs "alpha" namespaces with refresh "900000" and duration "300"
+  Then user sees the "alpha" namespace
+  And user sees selected graph duration "Last 5m"
+  And user sees selected graph refresh "Every 15m"
+
+@graph-page-toolbar
+Scenario: Open graph Tour
+  When user clicks graph tour
+  Then user "sees" graph tour
+
+@graph-page-toolbar
+Scenario: Close graph Tour
+  When user closes graph tour
+  Then user "does not see" graph tour
+
+@graph-page-toolbar
+Scenario: Open traffic dropdown
+  When user clicks graph traffic menu
+  Then user sees default graph traffic menu
+
+@graph-page-toolbar
+Scenario: Disable all traffic
+  When user disables all traffic
+  Then user does not see any traffic
+
+# todo: would be a better test if demos has tcp and/or grpc traffic
```

Comment:
We haven't had a reason to yet but this would be a good excuse to install mongodb with the demo apps.

Reply:
+1. I think that may actually be happening in order to align with CI.  If it does could go back and do some tcp testing.